testing/openjdk12/APKBUILD

```
@@ -0,0 +1,307 @@
+# Contributor: Simon Frankenberger <simon-alpine@fraho.eu>
+# Maintainer: Simon Frankenberger <simon-alpine@fraho.eu>
+pkgname=openjdk12
+pkgver=12.0.2_p10
+_pkgver=${pkgver/_p/+}
+pkgrel=0
+pkgdesc="Oracle OpenJDK 12"
+url="https://hg.openjdk.java.net/jdk-updates/jdk12u"
+arch="all !x86 !armhf !armv7" # openjdk11 is not yet available on 32 bit arches
+license="GPL-2.0 with Classpath"
+makedepends="autoconf
+bash
+gawk
+grep
+make
+openjdk11-jdk
+zip
+alsa-lib-dev
+cups-dev
+elfutils-dev
+fontconfig-dev
+freetype-dev
+giflib-dev
+lcms2-dev
+libexecinfo-dev
+libffi-dev
+libjpeg-turbo-dev
+libx11-dev
+libxext-dev
+libxrandr-dev
+libxrender-dev
+libxt-dev
+libxtst-dev
+linux-headers
+zlib-dev"
+depends="$pkgname-jmods $pkgname-demos $pkgname-doc $pkgname-jdk" # for the virtual openjdk12 package
+subpackages="$pkgname-jmods:_jmods:noarch
+$pkgname-demos:_demos:noarch
+$pkgname-doc:_doc:noarch
+$pkgname-dbg:_dbg
+$pkgname-jre:_jre
+$pkgname-src:_src:noarch
+$pkgname-jre-headless:_jre_headless
+$pkgname-jdk:_jdk"
+source="jdk-$_pkgver.tar.bz2::http://hg.openjdk.java.net/jdk-updates/jdk12u/archive/jdk-$_pkgver.tar.bz2
+
+build.patch
+aarch64.patch
+arm.patch
+ppc64le.patch
+x86.patch
+
+HelloWorld.java
+TestECDSA.java
+TestCryptoLevel.java
+Alpine_Bug_10126.java
```

```suggestion
	Alpine_Bug_10126.java
```

Please indent lists.